content/zh/docs/tasks/administer-cluster/kubeadm/kubeadm-certs.md

```diff
@@ -80,6 +80,11 @@ point to the CA certificate and key.
 否则, kubeadm 将独立运行 controller-manager，附加一个 `--controllers=csrsigner`
 的参数，并且指明 CA 证书和密钥。
 
+<!--[PKI certificates and requirements](/docs/setup/best-practices/certificates/) includes guidance on
```

Comment:
please leave `<!--` on a separate line so that in the future, we can easily detect upstream changes visually.